include/souper/Infer/AliveDriver.h

```diff
@@ -44,6 +44,8 @@ class AliveDriver {
   Inst *LHS, *PreCondition;
 
   Cache LExprCache, RExprCache;
+  std::map<const Inst *, std::string> NameMap;
+  void copyInputs(Cache& From, Cache &To, IR::Function &RHS);
```

Comment:
not that big of a deal, but you need to consistently put the & (or *) next to the name, not next to the type

Reply:
fixed